app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,223 @@
+/*
+* This is a manifest file that'll be compiled into application.css, which will include all the files
+* listed below.
+*
+* Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+* or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+*
+* You're free to add application-wide styles to this file and they'll appear at the bottom of the
+* compiled file so the styles you add here take precedence over styles defined in any styles
+* defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+* file per style scope.
+*
+*= require bootstrap_and_overrides
+*= require chosen
+*= require_tree .
+*= require_self
+*/
+
+//@import "bootstrap-sprockets";
+//@import "bootstrap";
+
+// Font Awesome
+@import "font-awesome-sprockets";
+@import "font-awesome";
+
+
+// Bootstrap body padding for fixed navbar
+body {
+  padding-top: 80px;
+  padding-bottom: 80px;
+}
+
+// @media (max-width: 1200px) {
+//   li .directory{
+//     margin-left: 70px !important;
+//     margin-right: 70px !important;
+//   }
+// }
+@media (max-width: 993px) {
+  .navbar-right.footer-text{
+    display:none;
+  }
+  // li .directory{
+  //   margin-left: 50px !important;
+  //   margin-right: 50px !important;
+  // }
+}
+@media (max-width: 767px) {
```

Comment:
Are these styles specific for Bootstrap? If so then please move it to `bootstrap_and_overrides.scss` file